lib/modules/showImages.js

```diff
@@ -644,286 +636,366 @@ function domainToModuleName(hostname) {
 	return domainStack.pop();
 }
 
-async function checkElementForImage(elem, selfTextExpando = false) {
-	if (module.options.hideNSFW.value) {
-		elem.NSFW = new Thing(elem).isNSFW();
-	} else {
-		elem.NSFW = false;
+async function convertGifToVideo(options) {
+	try {
+		const info = await ajax({
+			type: 'json',
+			url: '//upload.gfycat.com/transcodeRelease',
+			data: { fetchUrl: options.src },
+			cacheFor: DAY,
+		});
+
+		if (!info.gfyName) throw new Error();
+
+		return {
+			options: await gfycat.handleLink('', [undefined, null, info]),
```

Comment:
Maybe pass `info` as a third argument; I don't think it makes much sense for it to show up in the "regex results array"